### ISSUE TITLE
origins?

### ISSUE THREAD

**Issue by Brian151** [hqkirkland/DcrDecompressor#1]
I noticed that you starred + watched my project repo (thx, btw)

So, how did you happen upon it, and how/why did you get started in messing with the director/shockwave formats, anyways?

Just curious

**Comment by hqkirkland** [comment]
I come from a small community of developers that reverse engineer Habbo to make our own communities. Shockwave used to be what the game was based on :) ThuGie is also one of us. I dunno, reverse engineering is probably just in our blood.

Furthermore, I'm really interested in decompiling a game called Spineworld. I'd love to learn some if it's secrets, view its code, maybe inject some of my own. Date the shit out of it basically 😂

I could add functionality to the only version we have. Everyone wants 2009 but we only have 2008.

My project Whirlpool is a server for it.
http://dev.nodebay.com/ (use IE)

- Hunter

On Apr 29, 2017, at 3:54 AM, Brian <notifications@github.com<mailto:notifications@github.com>> wrote:


I noticed that you starred + watched my project repo (thx, btw)

So, how did you happen upon it, and how/why did you get started in messing with the director/shockwave formats, anyways?

Just curious

**Comment by Brian151** [comment]
oh, alright

sounds fun. some of my own goals pertain to this kind of stuff. I want to port a few games, and extract assets from dcrs.

yeah

alright

(meanwhile, I'm having fun configuring my haxe project to compile in a certain way...lol)

**Comment by hqkirkland** [comment]
Yeah man I'm just tryna get that game back! It's a bitch that Director was never sourced.

- Hunter

On Apr 29, 2017, at 12:41 PM, Brian <notifications@github.com<mailto:notifications@github.com>> wrote:


oh, alright

sounds fun. some of my own goals pertain to this kind of stuff. I want to port a few games, and extract assets from dcrs.

yeah

alright

(meanwhile, I'm having fun configuring my haxe project to compile in a certain way...lol)
